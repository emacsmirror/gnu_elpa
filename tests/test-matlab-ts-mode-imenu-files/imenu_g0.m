% -*- matlab-ts -*-
function ... % foo
    imenu_g0
    g1
    g2
    x = g3;
    [a, b] = g4(2);
end

function...
        a = ... % foo
        g1
    disp('in g1')
end

function ...
...
...%foo
        g2
    disp('in g2')
end

function x = ... % [a,b] =
        g3
    disp('in g3')
    function2 = 1;
    x = function2;
end

function [ ...
    a, ... % comment for a
    b  ... % comment for b
         ] ... 
         = ...
         g4(c)
    function g5
        disp('in g5')
    end

    disp('in g4')
    g5;
    a = c;
    b = c;
end
