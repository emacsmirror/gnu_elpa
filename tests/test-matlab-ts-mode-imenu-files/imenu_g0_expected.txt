g4.g5
g4
g3
g2
g1
imenu_g0
